### ISSUE TITLE
seacas missing files.

### ISSUE THREAD

**Issue by VictorEijkhout** [trilinos/Trilinos#3214]
A recent clone of the repository contains:
```
lrwxrwxrwx build/G-25072        0 2018-07-30 12:44 trilinos-git/packages/seacas/libraries/exodus/topology -> ../../../../docs/topology
lrwxrwxrwx build/G-25072        0 2018-07-30 12:44 trilinos-git/packages/seacas/libraries/ioss/html -> ../../../../docs/ioss_html
lrwxrwxrwx build/G-25072        0 2018-07-30 12:44 trilinos-git/packages/seacas/libraries/exodus/html -> ../../../../docs/html
```

All three don't exist. This trips up rpmbuild, which is what I use for making installations, since it checks that all claimed files actually exist.

Please fix?

Victor.

**Comment by jhux2** [comment]
@trilinos/seacas

**Comment by mhoemmen** [comment]
@gsjaardema 

Also, @DrBooom , you may have seen this when attempting the merge today, FYI.

**Comment by gsjaardema** [comment]
This should have been fixed with #3216
